omero/developers/Modules/Api.txt

```diff
@@ -73,15 +73,15 @@ Service Level 1 (direct database and Hibernate connections)
    disk space stats.
 -  RoiService
    :javadoc:`API <slice2html/omero/api/IRoi.html>`
-   working with ROIs.
+   working with ROIs. (now deprecated)
 -  ScriptService
    :javadoc:`API <slice2html/omero/api/IScript.html>`
    for uploading and launching Python scripts.
 -  SessionService
    :javadoc:`API <slice2html/omero/api/ISession.html>`
    for creating and working with OMERO sessions.
 -  ShareService
-   :javadoc:`API <slice2html/omero/api/IShare.html>`
+   :javadoc:`API <slice2html/omero/api/IShare.html>` (now deprecated)
```

Comment:
Ah, I figured they were associated with the explanatory text that followed (cf. ProjectionService).